Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Enrichment.yml

```diff
@@ -1149,9 +1151,6 @@ tasks:
                 value:
                   simple: '[\w-\.]+@[\w-]+\.+[\w-]{2,4}'
               unpack_matches: {}
-          - operator: uniq
```

Comment:
@capanw , did you leave out Unique on purpose?  I think that might be okay because if you have the same email from multiple sources, if it had unique I think that could cause issue with GridFieldSetup

Reply:
Thanks for pointing it out. That was miss. Added it back.